### PR TITLE
[Feature](information-schema) add SCHEMA_VERSION into information_schema.rowsets

### PR DESCRIPTION
**Pull request by BiteTheDDDDt**
## Proposed changes
add SCHEMA_VERSION into information_schema.rowsets
```sql
mysql [test]>select * from information_schema.rowsets where tablet_id=55182;
+------------+--------------------------------------------------+-----------+-----------------+--------+--------------+---------------+-------------+-----------------+----------------+---------------------+------------------------+----------------+
| BACKEND_ID | ROWSET_ID                                        | TABLET_ID | ROWSET_NUM_ROWS | TXN_ID | NUM_SEGMENTS | START_VERSION | END_VERSION | INDEX_DISK_SIZE | DATA_DISK_SIZE | CREATION_TIME       | NEWEST_WRITE_TIMESTAMP | SCHEMA_VERSION |
+------------+--------------------------------------------------+-----------+-----------------+--------+--------------+---------------+-------------+-----------------+----------------+---------------------+------------------------+----------------+
|      10009 | 020000000000005a714d2f4bc0290907ca9dc2c861aea6bb |     55182 |               0 |   8013 |            0 |             6 |           6 |               0 |              0 | 2024-06-04 16:43:36 | 2024-06-04 16:43:35    |              1 |
|      10009 | 0200000000000040714d2f4bc0290907ca9dc2c861aea6bb |     55182 |               0 |      0 |            0 |             5 |           5 |               0 |              0 | 2024-06-04 16:43:06 | 2024-06-03 15:23:56    |              0 |
|      10009 | 020000000000003e714d2f4bc0290907ca9dc2c861aea6bb |     55182 |               0 |      0 |            0 |             3 |           3 |               0 |              0 | 2024-06-04 16:43:06 | 2024-06-03 15:23:35    |              0 |
|      10009 | 020000000000003c714d2f4bc0290907ca9dc2c861aea6bb |     55182 |               0 |      0 |            0 |             2 |           2 |               0 |              0 | 2024-06-04 16:43:06 | 2024-06-03 15:23:04    |              0 |
|      10009 | 020000000000003f714d2f4bc0290907ca9dc2c861aea6bb |     55182 |               0 |      0 |            0 |             4 |           4 |               0 |              0 | 2024-06-04 16:43:06 | 2024-06-03 15:23:50    |              0 |
|      10009 | 020000000000003a714d2f4bc0290907ca9dc2c861aea6bb |     55182 |               0 |      0 |            0 |             0 |           1 |               0 |              0 | 2024-06-04 16:43:06 | 2024-06-03 15:22:36    |              0 |
+------------+--------------------------------------------------+-----------+-----------------+--------+--------------+---------------+-------------+-----------------+----------------+---------------------+------------------------+----------------+
```

### CODE REVIEW

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

File: be/src/exec/schema_scanner/schema_rowsets_scanner.cpp [hunk 1]
```diff
@@ -58,6 +58,7 @@ std::vector<SchemaScanner::ColumnDesc> SchemaRowsetsScanner::_s_tbls_columns = {
         {"DATA_DISK_SIZE", TYPE_BIGINT, sizeof(size_t), true},
         {"CREATION_TIME", TYPE_DATETIME, sizeof(int64_t), true},
         {"NEWEST_WRITE_TIMESTAMP", TYPE_DATETIME, sizeof(int64_t), true},
+        {"SCHEMA_VERSION", TYPE_INT, sizeof(int32_t), true},
```
Comment (gavinchou) [inline be/src/exec/schema_scanner/schema_rowsets_scanner.cpp:61]: make it bigint for flexibility

File: be/src/exec/schema_scanner/schema_rowsets_scanner.cpp [hunk 2]
```diff
@@ -242,6 +243,16 @@ Status SchemaRowsetsScanner::_fill_block_impl(vectorized::Block* block) {
         }
         RETURN_IF_ERROR(fill_dest_column_for_range(block, 11, datas));
     }
+    // SCHEMA_VERSION
+    {
+        std::vector<int32_t> srcs(fill_rowsets_num);
```
Comment (gavinchou) [inline be/src/exec/schema_scanner/schema_rowsets_scanner.cpp:248]: ditto, int64 for flexibility

**Review by gavinchou — CHANGES_REQUESTED** [review]

File: fe/fe-core/src/main/java/org/apache/doris/catalog/SchemaTable.java [hunk 3]
```diff
@@ -397,6 +397,7 @@ public class SchemaTable extends Table {
                             .column("DATA_DISK_SIZE", ScalarType.createType(PrimitiveType.BIGINT))
                             .column("CREATION_TIME", ScalarType.createType(PrimitiveType.DATETIME))
                             .column("NEWEST_WRITE_TIMESTAMP", ScalarType.createType(PrimitiveType.DATETIME))
+                            .column("SCHEMA_VERSION", ScalarType.createType(PrimitiveType.INT))
```
Comment (gavinchou) [inline fe/fe-core/src/main/java/org/apache/doris/catalog/SchemaTable.java:400]: ditto

File: be/src/exec/schema_scanner/schema_rowsets_scanner.cpp (hunk 2 above)
Comment (BiteTheDDDDt) [reply be/src/exec/schema_scanner/schema_rowsets_scanner.cpp:248]: 图片
schema version is int type field

**Review by gavinchou — APPROVED** [review]

**Review by zclllyybb — APPROVED** [review]

**Review by zhangstar333 — APPROVED** [review]

**Pull request merged**